src/overlays/actors/ovl_player_actor/z_player.c

```diff
@@ -353,11 +353,24 @@ void Player_Action_80850E84(Player* this, PlayState* play);
 void Player_Action_CsAction(Player* this, PlayState* play);
 
 // .bss part 1
+
+// For retail BSS ordering, the block number of sDogSpawnPos in Player_Update
+// must be between 0 and 53 inclusive.
+
+// TODO: There's probably a way to do this with less padding by spreading the variables out and moving
+// data around. It would be easier if we had more options for controlling BSS ordering in debug.
+#pragma increment_block_number 30
```

Comment:
for the record I am very much in favor of spreading data around near its localized usages, and have been slowly doing this in player when I can, as I document. 
I can make a more concentrated effort to do this all at once if it would be helpful

Reply:
The big problem with spreading the data around right now is that the block numbers don't work out for debug and we don't have a`#pragma increment_block_number` we can use there. Maybe it would work out better if the headers were split up more like they were in the original code?

Reply:
do you mean when we untangle our global headers in general? or something more specific to player?

Reply:
If the things that player needs are spread out in more headers, then more block numbers will be used (since each headers get assigned a block number if something is used from it) which might shift things enough in player so that the block numbers wrap around in the right place for spreading the data out to work. I don't remember the details for player though, like how many additional block numbers we need for debug

Reply:
okay I understand. Yeah we'll reevaluate as we do more header untangling